geode-core/src/main/java/org/apache/geode/internal/process/ControlFileWatchdog.java

```diff
@@ -73,12 +75,17 @@ public void run() {
   private void doWork() {
     try { // handle handle exceptions
       if (file.exists()) {
-        try { // always check stopAfterRequest after handleRequest
-          handleRequest();
-        } finally {
-          if (stopAfterRequest) {
-            stopMe();
+        // see if file is accessible
+        try (FileReader fileReader = new FileReader(file)) {
+          try { // always check stopAfterRequest after handleRequest
+            handleRequest(fileReader);
+          } finally {
+            if (stopAfterRequest) {
+              stopMe();
+            }
           }
+        } catch (FileNotFoundException e) {
+          e.printStackTrace();
```

Comment:
We shouldn't use `printStackTrace`. You should use logger to log it at debug or warn or something like that.

Reply:
Yes, good catch. Logging it now.